src/evo/netinfo.h

```diff
@@ -72,8 +73,9 @@ class NetInfoEntry
     bool operator!=(const NetInfoEntry& rhs) const { return !(*this == rhs); }
 
     template <typename Stream>
-    void Serialize(Stream& s) const
+    void Serialize(Stream& s_) const
```

Comment:
seems unrelated

Reply:
Not unrelated, the argument is changed to `s_` because it is overridden to require `CService` use ADDRV2 serialization and that overridden stream takes on the old variable name `s` (see below)

https://github.com/dashpay/dash/blob/8a9d24948d7783173b7a332e666c108770ce1ebe/src/evo/netinfo.h#L76-L78